Right now, if any exception occurs (except for "facade down"), we show an internal server error on the UI and set the status to "failed". For "facade down", we are keeping the current behavior as is.
